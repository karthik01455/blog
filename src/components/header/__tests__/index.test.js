import React from 'react';
import { render } from '@testing-library/react';
import Header from '..';

describe('Header', () => {
  it('Should rebder header when it is rendered', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});