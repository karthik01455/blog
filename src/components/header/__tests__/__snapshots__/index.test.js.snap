// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should rebder header when it is rendered 1`] = `
<DocumentFragment>
  <div
    class="body padding"
  >
    <h1>
      The Artifact
    </h1>
    <h2>
      <i>
        Culture & Art blog
      </i>
    </h2>
    <div
      class="nav"
    >
      <ul>
        <li>
          Blog
        </li>
        <li>
          About
        </li>
        <li>
          Contact
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
